keps/sig-apps/0008-sidecarcontainer.md

```diff
@@ -0,0 +1,148 @@
+---
+kep-number: 0008
+title: Sidecar Containers
+authors:
+  - "@joseph-irving"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-node
+reviewers:
+  - "@fetja"
```

Comment:
t and j are inverted

Reply:
@fejta oh dear, I'm sorry, that's a fail, let's pretend this never happened